devtools/support/debug.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+echo "=====================================HEADER====================================="
+echo "Run this script with your conda env activated"
+echo "Invoke the script like this: "
+echo "./debug.sh | tee -a debug.log"
```

Comment:
(not blocking) The `tee` command was unfamiliar to me - Is there a reason not to use the more familiar `>` (or `>&` to include STDERR)?

Also, might it be easier to use `source debug.sh` instead, since `./debug.sh` requires proper setting of execute permissions?

Reply:
This is copied from @mikemhenry's script but it seems like the point is that is dumps to both file and STDOUT: https://unix.stackexchange.com/a/709554

Reply:
`source debug.sh` does seem like an improvement